Library/Formula/libgraphqlparser.rb

```diff
@@ -0,0 +1,51 @@
+class Libgraphqlparser < Formula
+  desc "GraphQL query parser in C++ with C and C++ APIs"
+  homepage "https://github.com/graphql/libgraphqlparser"
+  url "https://github.com/graphql/libgraphqlparser/archive/v0.2.0.tar.gz"
+  sha256 "5064f63024c20cdc2c41970a6e9a5c7b053565db22f5f8dfb946923cb077f9de"
+
+  depends_on "cmake" => :build
+  depends_on "bison" => :recommended
```

Comment:
Would that make the dependency implicit? Do all targeted systems have bison and flex? The library comes with a checked in parser and lexer it fallsback to using if the appropriate version of bison and flex aren't present. That's why I was thinking `recommended` made the most sense.

https://github.com/graphql/libgraphqlparser#building-libgraphqlparser